stability/runPythonOnWindows.bat

```diff
@@ -1,11 +1,11 @@
-py %WORKSPACE%\stability\stability\windows_native-stability-test.py --stabilization --iterations=10
+START "CORECLR_PERF_RUN" /B /WAIT /HIGH /AFFINITY 0x2 py %WORKSPACE%\stability\stability\windows_native-stability-test.py --iterations=11
```

Comment:
I think we should have this inside the stability test wrapper, just when we launch the actual process.